plutus-playground-lib/src/Playground/Contract.hs

```diff
@@ -34,11 +38,20 @@ import           Data.Swagger                (Schema, ToSchema)
 import           GHC.Generics                (Generic)
 import           Playground.API              (FunctionSchema)
 import           Playground.Interpreter.Util
-import           Playground.TH               (mkFunction, mkFunctions, mkSingleFunction)
+import           Playground.TH               (mkFunction, mkFunctions, mkKnownCurrencies, mkSingleFunction)
 import           Wallet.API                  (payToPublicKey_)
 import           Wallet.Emulator             (addBlocksAndNotify, runWalletActionAndProcessPending)
 import           Wallet.Emulator.Types       (MockWallet, Wallet (..))
 
+-- TODO: Just an example for debugging, we should move this into a test
+import           Data.List.NonEmpty          (NonEmpty ((:|)))
+import           Ledger.Validation           (ValidatorHash (..))
+import           Playground.API              (KnownCurrency (..), TokenId (..))
+
+c :: KnownCurrency
+c = KnownCurrency (ValidatorHash "") "MyC" (TokenId "" :| [])
```

Comment:
Can you give it a `TokenId` that's non-empty? Technically they're allowed to be empty but I don't think that should be the standard case for debugging since you'll need to display the name of the token at some point

Reply:
I was actually wondering about empty token ids. We probably want a convention on what the token id is for a "normal" currency, since otherwise that's annoying additional information you need to know to use it. Maybe it should be the same as the currency id in the case where you don't care?

Reply:
I've moved this into a test anyway and given the token a proper string

Reply:
> Maybe it should be the same as the currency id in the case where you don't care?

If there's only one token ID then we don't need to show it to the user at all because there is no choice to be made

Reply:
Sure, but I still need to know it in order to work with the currency. I guess it's fine when you're only transferring things, which is most of the time, since all the stuff already has IDs. It's probably only an issue with forging,  and then the MPS can enforce it being whatever.

Reply:
the empty string would be the cheapest token ID (ie. it would add the least amount to every transaction that handles it) so maybe that will turn out to be the default